java/arcs/core/data/expression/diagram.uml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does it server any purpose?

Reply:
accidental click in IntelliJ created this file on disk (I think it's 'Show Diagram')
Will remove